File: 100-Days-Series/Day 67/AlphabetPalindromicPattern.java
class AlphabetPalindromicPattern{
    // Fucntion to print Palindromic Pattern with Alphabet
    public static void getPrintPalindromicNumber(int rows){
        for(int i=1;i<=rows;i++){
            char ch='A';
            // Print Spaces
            for(int j=1;j<=rows-i;j++){
                System.out.print("  ");
            }
            // Print Increasing Alphabet
            for(int j=1;j<=i;j++){
                System.out.print((ch++)+" ");
            }
            for(int j=i-1;j>=1;j--){
                System.out.print((ch++)+" ");
            }
            System.out.println();
        }
    }
    // Main Method 
    public static void main(String [] args){
        int rows;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of rows : ");
        rows=sc.nextInt();
        getPrintPalindromicNumber(rows);    
    }
}

File: 100-Days-Series/Day 67/AlphabetPalindromicPattern.c
#include<stdio.h>
// Funtion to print Palindromic Pattern with Alphabet
void getPrintPalindromicPattern(int rows){
    for(int i=1;i<=rows;i++){
        char ch='A';
        // Print Spaces
        for(int j=1;j<=rows-i;j++){
            printf("  ");
        }
        // Print Increasing Alphabet
        for(int j=1;j<=i;j++){
            printf("%c ",ch++);
        }
        for(int j=i-1;j>=1;j--){
            printf("%c ",ch++);
        }
        printf("\n");
    }
}
// Let's Start Here
void main(){
    int rows; 
    printf("\nEnter the number of rows : ");
    scanf("%d",&rows);
    getPrintPalindromicPattern(rows);
}